apisix/core/config_yaml.lua

```diff
@@ -395,6 +430,14 @@ function _M.new(key, opts)
         key = sub_str(key, 2)
     end
 
+    if is_use_admin_api() then
+        if item_schema and item_schema.properties then
+            -- allow clients to specify modifiedIndex to control resource changes.
+            item_schema.properties.modifiedIndex = {
+                type = "integer",
+            }
+        end
+    end
```

Comment:
It seems that this should also be implemented in standalone. Should we be concerned that the schema is permanently modified, i.e. should we modify a deepcopy?

Reply:
fixed.